local-cli/generator-android/templates/src/app/build.gradle

```diff
@@ -24,6 +24,14 @@ import com.android.build.OutputFile
  *   // whether to bundle JS and assets in release mode
  *   bundleInRelease: true,
  *
+ *   // whether to have JS in dev mode for a variant (with warnings)
+ *   devModeFor[VARIANT]: true|false,
+ *   (default : true only if variant contains "release")
+ *   Note: this param only apply if JS files are bundled.
+ *
+ *   // whether to bundle JS and assets in release mode
+ *   bundleInRelease: true,
```

Comment:
Looks like this is now present twice?

     // whether to bundle JS and assets in release mode
     bundleInRelease: true,